inc/form.class.php

```diff
@@ -646,8 +646,12 @@ public function showFormList($rootCategory = 0, $keywords = '', $helpdeskHome =
             'contains' => $keywords
       ]);
       if ($selectedCategories != '') {
-         $query_faqs = "SELECT * FROM ($query_faqs)  AS `faqs`
+         $query_faqs = "SELECT `faqs`.* FROM ($query_faqs)  AS `faqs`
          WHERE `faqs`.`knowbaseitemcategories_id` IN (SELECT `knowbaseitemcategories_id` FROM `$table_cat` WHERE `id` IN ($selectedCategories) AND `knowbaseitemcategories_id` <> '0')";
+      } else {
+         $query_faqs = "SELECT `faqs`.* FROM ($query_faqs)  AS `faqs`
+         LEFT JOIN `$table_cat` ON (`faqs`.`knowbaseitemcategories_id` = `$table_cat`.`knowbaseitemcategories_id`)
```

Comment:
after some review by @flegastelois, i think we have an issue in this query.

i think we must have an INNER JOIN or exclude lines by WHERE knowbaseitemcategories_id IS NOT NULL